reflect_client.go

```diff
@@ -44,7 +44,17 @@ func NewDefaultFileDescriptorSetServiceClient(token string) reflectv1beta1connec
 	}
 	return reflectv1beta1connect.NewFileDescriptorSetServiceClient(
 		http.DefaultClient, "https://buf.build",
-		connect.WithInterceptors(NewAuthInterceptor(token)),
+		connect.WithInterceptors(
+			NewAuthInterceptor(token),
+			connect.UnaryInterceptorFunc(func(call connect.UnaryFunc) connect.UnaryFunc {
+				return func(ctx context.Context, req connect.AnyRequest) (connect.AnyResponse, error) {
+					// decorate user-agent with the name of the package
+					userAgent := fmt.Sprintf("%s prototransform-go", req.Header().Get("User-Agent"))
```

Comment:
Unfortunately not. There's no easy way to get something like a SHA. And since it's a library, we don't try to maintain a constant in the code with a version 🤷 
(It could possibly be done via `-X` ldflags when building, but we don't have control over that when _other_ people build this to link it into their binary.)

Reply:
We could try to add something later if we decide it's important to know the version -- like a manually-maintained constant that we keep in sync w/ release tags.

Reply:
Thanks, more of a shower thought. Not worth the effort (at least for now) imo.